sql/parser/ast.go

```diff
@@ -704,43 +754,82 @@ func (node *Subquery) String() string {
 	return fmt.Sprintf("(%v)", node.Select)
 }
 
+// BinaryOp represents a binary operator.
+type BinaryOp int
+
+// BinaryExpr.Operator
+const (
+	Bitand BinaryOp = iota
+	Bitor
+	Bitxor
+	Plus
+	Minus
+	Mult
+	Div
+	Mod
+	Concat
+)
+
+var binaryOpName = [...]string{
+	Bitand: "&",
+	Bitor:  "|",
+	Bitxor: "^",
+	Plus:   "+",
+	Minus:  "-",
+	Mult:   "*",
+	Div:    "/",
+	Mod:    "%",
+	Concat: "||",
```

Comment:
Yep, "concat" is what ANSI specifies. Do you see value in supporting logical or via some mode setting?